### PR TITLE
feat-604: Added input to change cohort in Admin Panel

### LINKED ISSUE #604: feature: Should be able to assign cohort from the admin dashboard
Screenshot 2024-08-31 at 12 36 53 AM

Cohort 2/3

### PR DESCRIPTION
**Pull request by ItsFlash10**
### PR Fixes:

- Admin would be able to change the cohort for any uploaded track.

@hkirat @devsargam 

Resolves #604 

https://github.com/user-attachments/assets/c492396b-3ba3-4501-a610-44ac521a62e8

### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

**Comment by ItsFlash10** [comment]
PS: UI can be improved as well but I've catered the functionality atm as it's important as highlighted a couple of days ago for the filters.

File: apps/web/app/admin/layout.tsx
Comment (devsargam) [inline apps/web/app/admin/layout.tsx:1]: Just a suggestion from my side. Try to turn off your preferred formatting tool and try to minimise the diff.
image

**Review by devsargam — APPROVED** [review]

**Comment by devsargam** [comment]
Testing locally

File: apps/web/app/admin/layout.tsx
Comment (ItsFlash10) [reply apps/web/app/admin/layout.tsx:1]: > Just a suggestion from my side. Try to turn off your preferred formatting tool and try to minimise the diff. image

Will do that, it's there actually as I added `prettier-tailwind-plugin` in the last PR as it was missing in this project and I think I added a log or something there and later removed it but the classes got sorted coz of the plugin.

Comment (devsargam) [reply apps/web/app/admin/layout.tsx:1]: Cool, but usually formatting change will be a separate pr in itself and happens when the repo's pr count is almost 0. Good thing to keep in mind tho.

Comment (ItsFlash10) [reply apps/web/app/admin/layout.tsx:1]: Yes ser

**Comment by devsargam** [comment]
This updates the backend but can we get the filter functionality working too?

**Review by devsargam — CHANGES_REQUESTED** [review]
Make the filter option working as well

**Comment by ItsFlash10** [comment]
> Make the filter option working as well

Done

**Review by devsargam — APPROVED** [review]

**Comment by devsargam** [comment]
Testing locally tomorrow

**Comment by devsargam** [comment]
works perfectly

**Comment by devsargam** [comment]
Thank u @ItsFlash10. You will receive the bounty later

**Pull request merged**